Documentation/host-local.md

```diff
@@ -5,30 +5,41 @@ it can include a DNS configuration from a `resolv.conf` file on the host.
 
 ## Overview
 
-host-local IPAM plugin allocates IPv4 addresses out of a specified address range.
+host-local IPAM plugin allocates ip addresses out of a set of address ranges.
 It stores the state locally on the host filesystem, therefore ensuring uniqueness of IP addresses on a single host.
 
 ## Example configurations
 
-IPv4:
 ```json
 {
 	"ipam": {
 		"type": "host-local",
-		"subnet": "10.10.0.0/16",
-		"rangeStart": "10.10.1.20",
-		"rangeEnd": "10.10.3.50",
-		"gateway": "10.10.0.254",
+		"ranges": [
```

Comment:
Hah! Funny.
I thought about the same thing, but ultimately decided on the keeping the same names for ease of transition.